article/sections/_3_graph_overview.tex

```diff
@@ -0,0 +1,160 @@
+directed acyclic graph (DAG) that encodes an analyst's knowledge and belief
+about the data generation process of the problem at hand, can a specific
+causal effect be identified?
+As such, we emphasize that DAGs are great tools
+for a modeler to encode their assumptions about a problem, but not necessarily
+a guide on how to estimate a causal effects of interest.
+
+\subsection{Prior Uses of Causal Graphs in Choice Modelling}
+\label{sec:choice-graphs}
+
+In our last subsection, we reviewed the basics of causal graphs: what are they and why are they useful?
+We targeted that subsection at choice modellers who are unfamiliar with these tools.
+However, readers should be aware of the history of causal graphs in choice modelling.
+Indeed, choice modellers have used causal graphs (in limited fashion) for years.
+Here are three examples to illustrate our point.
+First, consider the case of Random Utility Maximization (RUM) models.
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.25\textwidth]{rum-causal-graph}
+   \caption{Archetypical RUM causal diagram}
+   \label{fig:example-graph-rum}
+\end{figure}
+
+For decades, choice modellers have used stylized DAGs to depict RUM models.
+In particular, these diagrams illustrate an assumed choice-making (i.e., causal) process.
+As an example, see Figure \ref{fig:example-graph-rum}, reproduced from Figure 1 of \citet{ben_2002_integration}.
+\citeauthor{ben_2002_integration} draw a two-part process.
+First, they assume that explanatory variables cause unobserved utilities.
+Then, they assume that the unobserved utilities cause the choice.
+
+Note that although RUM diagrams adequately show the choice process, we still call them stylized.
+Specifically, these diagrams lack detail about the relationships between explanatory variables.
+When speaking collectively, we cannot tell if one explanatory variable causes another.
+Unfortunately, as shown in Section \ref{sec:graph-importance}, such knowledge is crucial.
+Without more detailed causal knowledge, our inferences may be inconsistent and arbitrarily bad.
+
+\begin{figure}
+   \centering
+   \includegraphics[width=0.5\textwidth]{iclv-causal-graph}
+   \caption{Archetypical ICLV causal diagram}
+   \label{fig:example-graph-iclv}
+\end{figure}
+
+
+Besides RUM models, causal graphs appear in the literature on Integrated Choice and Latent Variable (ICLV) models.
+An example of an ICLV model is in Figure \ref{fig:example-graph-iclv}, based on Figure 5 of \citet{ben_2002_integration}.
+As with RUM models, ICLV causal graphs represent assumptions about the choice process.
+Here concern typically centers around unobserved mediators.
+These unobserved variables cause the outcome and are caused by one's observed covariates.
+Omitting such mediators leads to models that misrepresent the assumed choice process.
+As a result, researchers care deeply about ICLV models that avoid these behavioral misrepresentations.
```

Comment:
I'm confused about what this is the result of? The result of the misrepresentation of the choice process? Or something else?